db/tasks.go

```diff
@@ -229,12 +241,12 @@ func (db *DB) CreateBackupTask(owner string, job *Job) (*Task, error) {
 		`INSERT INTO tasks
 		    (uuid, owner, op, job_uuid, status, log, requested_at,
 		     store_uuid, store_plugin, store_endpoint,
-			 target_uuid, target_plugin, target_endpoint, restore_key, 
+			 target_uuid, target_plugin, target_endpoint, restore_key,
```

Comment:
Can you remove these spurious whitespace changes from your commit?  We can make a cleanup commit later, but they don't really belong in this changeset.

Reply:
Sorry about that, that's what I get for not using `git diff`.